src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java

```diff
@@ -422,57 +245,56 @@ public void getFixedFieldsOfVisualMaterial_return200Status() {
     String url = getURI(FIXED_FIELDS_CODE_GROUPS_URL);
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "006")
-      .param("headerTypeCode", "22")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "006").param("headerTypeCode", "22")
+        .param("lang", "ita").headers(headers).when().get(url).then().statusCode(200);
   }
 
-
-
-
   @Test
   public void getFixedFieldsByTag008_return200Status() {
 
     String url = getURI(FIXED_FIELDS_CODE_GROUPS_URL);
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "008")
-      .param("headerTypeCode", "31")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "008").param("headerTypeCode", "31")
+        .param("lang", "ita").headers(headers).when().get(url).then().statusCode(200);
   }
 
   public void getFixedFieldCodesGroups_failed() {
 
     String url = getURI(FIXED_FIELDS_CODE_GROUPS_URL);
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "009")
-      .param("headerTypeCode", "1")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "009").param("headerTypeCode", "1")
+        .param("lang", "ita").headers(headers).when().get(url).then().statusCode(200);
   }
 
+  @Test
+  public void getAuthorityFixedFieldsFromLeaderReturn200Status() {
 
+    String url = getURI(AUT_FIXED_FIELDS_CODE_GROUPS_URL);
+    Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
+
+    given().param("leader", AUTHORITY_LEADER).param("code", "000").param("headerTypeCode", "9").param("lang", "eng")
+        .headers(headers).when().get(url).then().statusCode(200);
+  }
+
+  @Test
+  public void getAuthorityFixedFieldsByTag008Return200Status() {
+
+    String url = getURI(AUT_FIXED_FIELDS_CODE_GROUPS_URL);
+    Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
+    given().param("leader", AUTHORITY_LEADER).param("code", "008").param("headerTypeCode", "10").param("lang", "eng")
+        .headers(headers).when().get(url).then().statusCode(200);
+  }
+
+  public void getAuthorityFixedFieldCodesGroupsFailed() {
```

Comment:
is the @test annotation omitted intentionally?

Reply:
I have forgotten. Sorry